tests/unit/services/unified-event-handler-test.js

```diff
@@ -205,7 +210,7 @@ test('unregister destroys the DOM handler for an event after all callbacks have
 
   let callback1Spy = sandbox.stub();
```

Comment:
no need to suffix this with extra words. `callback1` and `callback2` are sufficient (but because over specificity is not required and because in-this case they are not spies).

Reply:
Fixed here, and a couple of other places. For these PR's, I figured it's better to touch as little existing code as I can, to keep the PR cleaner. Happy to come back in and do some spring cleaning if you think that's necessary.